packages/components/src/Toggle/LabelToggle/LabelToggle.scss

```diff
@@ -0,0 +1,49 @@
+@mixin radio-toolbar-input($color, $opacity) {
+	transition: 0.2s ease-out;
+	color: $color;
+	opacity: $opacity;
+}
+
+.radio-toolbar {
+	display: inline-flex;
+	background-color: $gallery;
+	border-radius: $padding-larger;
+	margin-right: $padding-small;
+	box-shadow: 0px 1px 3px 0px rgba(0, 0, 0, 0.25) inset;
+}
+
+.radio-toolbar input[type='radio'] {
+	display: none;
+}
+
+.radio-toolbar label {
+	background-color: $gallery;
+	padding: 0 8px;
+	border-radius: $padding-larger;
+	color: $black;
+	font-weight: $font-weight-semi-bold;
+	margin: 2px 0;
+
+	&:first-of-type {
+		margin: 2px 0 2px 2px;
+	}
+
+	&:last-of-type {
+		margin: 2px 2px 2px 0;
+	}
+}
+
+.radio-toolbar label:hover {
+	background-color: $scooter;
+	color: $white;
+}
+
+.radio-toolbar input[type='radio'] + label {
+	@include radio-toolbar-input($black, 0.54);
+}
+
+.radio-toolbar input[type='radio']:checked + label {
+	@include radio-toolbar-input($white, 1);
+	background-color: $lochmara;
+	font-weight: $font-weight-semi-bold;
+}
```

Comment:
What about the focus state?

Reply:
I really don't know how to handle the autofocus state.
Not only as it's not present in the guidelines, but also technically cause we have the focus on the hidden radio buttons and we should highlight the containers of theses.
Could we use ally-focus-within to know it ? not sure at all it's the right way to do it

Reply:
You can try using `.tc-label-toggle:focus-within { }`

Reply:
Not sure it would work as intented with :focus-within as it will always show the focus highlight even if we don't navigate in the form with a keyboard